README.md

```diff
@@ -3,9 +3,13 @@
 
 - [Stacks Webapp Template](https://github.com/amido/stacks-webapp-template)
 - [Stacks .NET](https://github.com/amido/stacks-dotnet)
-- [Stacks JAVA](https://github.com/amido/stacks-java)
+- [Stacks JAVA with CQRS](https://github.com/amido/stacks-java)
+- [Stacks JAVA](https://github.com/amido/stacks-java/tree/feat/api-with-no-cqrs)
```

Comment:
Stylistically shouldn't Java be spelt "Java" and not "JAVA"?

Reply:
Stylistcally yes